src/http/server.rs

```diff
@@ -57,6 +61,22 @@ async fn entity_validator(
     }
 }
 
+fn build_cors(ayb_cors: Option<AybConfigCors>) -> Cors {
+    let mut cors = Cors::default().allow_any_header().allow_any_method();
+
+    if ayb_cors
+        .as_ref()
+        .is_some_and(|conf| conf.origin.trim() == "*")
+        || ayb_cors.is_none()
+    {
+        cors = cors.allow_any_origin()
+    } else {
+        cors = cors.allowed_origin(ayb_cors.as_ref().unwrap().origin.trim());
```

Comment:
Are we guaranteed the `unwrap` is safe here?

Reply:
It is safe because of the `is_none` check above, I'm going to change this anyway